tabs/auxiliary.js

```diff
@@ -112,13 +119,11 @@ TABS.auxiliary.initialize = function (callback) {
             var rangeElement = $(this).data('rangeElement');
             rangeElement.remove();
         });
-        
-        $(rangeElement).find('.channel').val(auxChannelIndex);
 
+        //$(rangeElement).find('.channel').val(auxChannelIndex);
```

Comment:
If this is not needed any more it has to be removed. Leaving in commented out code (especially without annotation) is just cluttering up the code base.

But I suspect that this is actually needed. Currently, after setting and saving the modes:

![image](https://user-images.githubusercontent.com/4742747/31866253-68afc520-b7d9-11e7-84b9-f11270b44feb.png)
 
when reloading the modes tab, all modes show as 'AUTO SELECT':

![image](https://user-images.githubusercontent.com/4742747/31866259-8415d782-b7d9-11e7-9d0b-4ebcfed8b933.png)